docs/index.rst

```diff
@@ -191,33 +191,10 @@ Post-installation instructions to use Docker without root privileges (logout/log
    $ sudo groupadd docker
    $ sudo usermod -aG docker $USER
 
-To start an Elasticsearch container, you can use the
-``misc/dc`` script wrapper on top of ``docker-compose``::
+Test your docker installation with
 
-   $ misc/dc up -d
+    $ docker run hello-world
```

Comment:
I would remove the container afterward: `docker run --rm hello-world`.

Reply:
done!